FreydCategoriesForCAP/gap/AdditiveClosure.gi

```diff
@@ -1356,6 +1362,256 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_ADDITIVE_CLOSURE,
         
     fi;
     
+    if HasIsMonoidalCategory( underlying_category ) and IsMonoidalCategory( underlying_category ) then
+        
+        ##
+        AddTensorUnit( category,
+          function( cat )
+            
+            return ObjectConstructor( cat, [ TensorUnit ( UnderlyingCategory( cat ) ) ] );
+            
+          end );
+        
+        ##
+        AddTensorProductOnObjects( category,
+          function( cat, obj_1, obj_2 )
+            local underlying_category, objdat_1, objdat_2, len_1, len_2;
+            
+            underlying_category := UnderlyingCategory( cat );
+            objdat_1 := ObjectDatum( cat, obj_1 );
+            objdat_2 := ObjectDatum( cat, obj_2 );
+            len_1 := Length( objdat_1 );
+            len_2 := Length( objdat_2 );
+            
+            return ObjectConstructor( cat,
```

Comment:
The existing code in AdditiveClosure uses `AdditiveClosureObject` instead of `ObjectConstructor` (and `AdditiveClosureMorphism` instead of `MorphismConstructor`) -> I suggest to keep this consistent.

Reply:
I remember telling @damian-delafuente that I expect you to suggest exactly these changes 😄

Reply:
done